packages/components/nfts/NFTs.tsx

```diff
@@ -66,6 +66,7 @@ export const NFTs: React.FC<{
           )}
           minElemWidth={250}
           gap={layout.spacing_x2}
+          noFixedHeight={Platform.OS !== "web"}
```

Comment:
this will load up to 10000 nfts, we can't do this here!

Reply:
ok